var/spack/repos/builtin/packages/ocamlbuild/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ocamlbuild(MakefilePackage):
+    """OCamlbuild is a generic build tool,
+       that has built-in rules for building OCaml library and programs."""
+
+    # Add a proper url for your package's homepage here.
+    homepage = "https://ocaml.org/learn/tutorials/ocamlbuild/"
+    url      = "https://github.com/ocaml/ocamlbuild/archive/master.zip"
+    git      = "https://github.com/ocaml/ocamlbuild"
+    # list_url = "https://github.com/ocaml/ocamlbuild/releases/tag"
+    # list_url = "https://github.com/ocaml/ocamlbuild/archive/"
+
+    # Add a list of GitHub accounts to
+    # notify when the package is updated.
+    maintainers = ['gasche']
+
+    # Add proper versions here.
+    # spack -d install --no-checksum ocamlbuild
+    version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```

Reply:
This isn't technically needed -- since you can `spack install --no-checksum ocamlbuild@<any-version-or-branch>` -- but it does document the available branch.